### PR TITLE
Add basic concepts

### PR DESCRIPTION
**Pull request by songyouwei**
add missing concept docs for `Variable` `Tensor` `LoDTensor` `Operator` `Program` `Executor`

### CODE REVIEW

**Comment by songyouwei** [comment]
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_tensor html (5)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_tensor html (1)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_tensor html (2)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_tensor html (3)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_tensor html (4)

**Comment by songyouwei** [comment]
10 88 157 35_8080_documentation_docs_zh_beginners_guide_index_cn html (2)
10 88 157 35_8080_documentation_docs_zh_beginners_guide_index_cn html (1)
10 88 157 35_8080_documentation_docs_zh_beginners_guide_index_cn html

**Comment by songyouwei** [comment]
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (7)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (6)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (5)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (4)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (3)
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (2)

File: doc/fluid/user_guides/howto/basic_concept/index_cn.rst [hunk 1]
```diff
@@ -1,374 +1,23 @@
...
-更高阶的应用案例，请参考 `模型库 <../../../user_guides/models/index_cn.html>`_ 中的相关内容。
+############
+基本概念
+############
+
+本文介绍 Fluid 中的基本概念：
+
+- `Variable <variable.html>`_ : Variable表示变量，在Fluid中可以包含任何类型的值，在大多数情况下是一个Lod-Tensor。
+- `Tensor <tensor.html>`_ : Tensor表示数据。
+- `LoD-Tensor <lod_tensor.html>`_ : LoD-Tensor是Fluid中特有的概念，它在Tensor基础上附加了序列信息，支持处理变长数据。
```
Comment (phlrain) [inline doc/fluid/user_guides/howto/basic_concept/index_cn.rst]: 是Paddle的高级特性

Comment (songyouwei) [reply doc/fluid/user_guides/howto/basic_concept/index_cn.rst]: done

**Comment by songyouwei** [comment]
10 88 157 35_8080_documentation_docs_zh_user_guides_howto_basic_concept_index_cn html (8)

**Comment by songyouwei** [comment]
10 88 157 35_8080_documentation_docs_zh_advanced_usage_index_cn html

**Review by lanxianghit — APPROVED** [review]

**Review by phlrain — APPROVED** [review]

**Pull request merged**